### PR TITLE
Remove rewards feature

### LINKED ISSUE #640: Remove Rewards Feature
There are no expectations to use this feature anymore. Now that we are refactoring, it's a good opportunity to remove it.

### PR DESCRIPTION
**Pull request by amonsosanz**
Closes #640.

### What does this PR does?

This PR removes the airdrop/rewards feature from the wallet.

### How to test?

Just make sure the app works normally and rewards are no longer there.

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Respect code style and lint
- [x] Update documentation (if needed)

### CODE REVIEW

File: src/views/token-swap/components/quote-selector/quote-selector.styles.js [hunk 1]
```diff
@@ -22,9 +22,6 @@ const useQuoteSelectorStyles = createUseStyles((theme) => ({
   toTokensHeader: {
     flex: "1 1 40%",
   },
-  rewardHeader: {
-    flex: "1 1 20%",
```
Comment (adrifdez) [inline src/views/token-swap/components/quote-selector/quote-selector.styles.js:26]: Token swap use different rewards. They are not related to this issue

File: src/views/token-swap/components/quote-selector/quote-selector.view.jsx [hunk 2]
```diff
@@ -17,7 +17,6 @@ function QuoteSelector({
       <div className={classes.header}>
         <p className={classes.nameHeader}>Liquidity provider</p>
         <p className={classes.toTokensHeader}>You get (ETH)</p>
-        <p className={classes.rewardHeader}>Reward</p>
```
Comment (adrifdez) [inline src/views/token-swap/components/quote-selector/quote-selector.view.jsx:20]: Token swap use different rewards. They are not related to this issue

File: src/views/token-swap/components/quote/quote.styles.js [hunk 3]
```diff
@@ -72,22 +72,6 @@ const useQuoteStyles = createUseStyles((theme) => ({
...
     marginTop: theme.spacing(1),
   },
-  rewardCell: {
-    flex: "1 1 20%",
-    display: "flex",
-    alignItems: "center",
-    justifyContent: "center",
-    flexDirection: "column",
-  },
-  reward: {
```
Comment (adrifdez) [inline src/views/token-swap/components/quote/quote.styles.js:82]: Token swap use different rewards. They are not related to this issue

File: src/views/token-swap/components/quote/quote.view.jsx [hunk 4]
```diff
@@ -44,19 +44,6 @@ function Quote({
             </p>
           )}
         </div>
-        <div className={classes.rewardCell}>
-          {info.rewards ? (
```
Comment (adrifdez) [inline src/views/token-swap/components/quote/quote.view.jsx:48]: Token swap use different rewards. They are not related to this issue

File: src/views/token-swap/components/selected-quote/selected-quote.style.js [hunk 5]
```diff
@@ -61,16 +61,6 @@ const useSelectedQuoteStyles = createUseStyles((theme) => ({
     color: theme.palette.grey.main,
     fontWeight: theme.fontWeights.bold,
   },
-  reward: {
-    borderTop: `1px solid ${theme.palette.grey.veryLight}`,
```
Comment (adrifdez) [inline src/views/token-swap/components/selected-quote/selected-quote.style.js:65]: Token swap use different rewards. They are not related to this issue

File: src/views/token-swap/components/selected-quote/selected-quote.view.jsx [hunk 6]
```diff
@@ -22,22 +22,6 @@ function SelectedQuote({
 }) {
   const classes = useSelectedQuoteStyles();
 
-  function getRewardAmountInTokens() {
-    return getFixedTokenAmount(
```
Comment (adrifdez) [inline src/views/token-swap/components/selected-quote/selected-quote.view.jsx:26]: Token swap use different rewards. They are not related to this issue

File: src/views/token-swap/components/quote-selector/quote-selector.styles.js [hunk 7]
```diff
@@ -22,9 +22,6 @@ const useQuoteSelectorStyles = createUseStyles((theme) => ({
   toTokensHeader: {
     flex: "1 1 40%",
   },
-  rewardHeader: {
```
Comment (AlbertoElias) [inline src/views/token-swap/components/quote-selector/quote-selector.styles.js:25]: this is from token swap, so it should stay. It's' a different kind of reward

File: src/views/token-swap/components/quote-selector/quote-selector.view.jsx (hunk 2 above)
Comment (AlbertoElias) [inline src/views/token-swap/components/quote-selector/quote-selector.view.jsx:20]: this is from token swap, so it should stay. It's' a different kind of reward

File: src/views/token-swap/components/quote/quote.styles.js [hunk 8]
```diff
@@ -72,22 +72,6 @@ const useQuoteStyles = createUseStyles((theme) => ({
     color: theme.palette.red.main,
     marginTop: theme.spacing(1),
   },
-  rewardCell: {
```
Comment (AlbertoElias) [inline src/views/token-swap/components/quote/quote.styles.js:75]: this is from token swap, so it should stay. It's' a different kind of reward

**Review by adrifdez — CHANGES_REQUESTED** [review]

File: src/views/token-swap/components/quote/quote.view.jsx [hunk 9]
```diff
@@ -44,19 +44,6 @@ function Quote({
             </p>
           )}
         </div>
-        <div className={classes.rewardCell}>
```
Comment (AlbertoElias) [inline src/views/token-swap/components/quote/quote.view.jsx:47]: this is from token swap, so it should stay. It's' a different kind of reward

File: src/views/token-swap/components/selected-quote/selected-quote.style.js [hunk 10]
```diff
@@ -61,16 +61,6 @@ const useSelectedQuoteStyles = createUseStyles((theme) => ({
     color: theme.palette.grey.main,
     fontWeight: theme.fontWeights.bold,
   },
-  reward: {
```
Comment (AlbertoElias) [inline src/views/token-swap/components/selected-quote/selected-quote.style.js:64]: this is from token swap, so it should stay. It's' a different kind of reward

File: src/views/token-swap/components/selected-quote/selected-quote.view.jsx [hunk 11]
```diff
@@ -22,22 +22,6 @@ function SelectedQuote({
 }) {
   const classes = useSelectedQuoteStyles();
 
-  function getRewardAmountInTokens() {
```
Comment (AlbertoElias) [inline src/views/token-swap/components/selected-quote/selected-quote.view.jsx:25]: this is from token swap, so it should stay. It's' a different kind of reward

File: src/views/token-swap/components/quote-selector/quote-selector.view.jsx (hunk 2 above)
Comment (amonsosanz) [reply src/views/token-swap/components/quote-selector/quote-selector.view.jsx]: Thx! Reverted!

Comment (amonsosanz) [reply src/views/token-swap/components/quote-selector/quote-selector.view.jsx]: Thx! Reverted!

File: src/views/token-swap/components/quote-selector/quote-selector.styles.js (hunk 7 above)
Comment (amonsosanz) [reply src/views/token-swap/components/quote-selector/quote-selector.styles.js]: Thx! Reverted!

File: src/views/token-swap/components/quote-selector/quote-selector.styles.js (hunk 1 above)
Comment (amonsosanz) [reply src/views/token-swap/components/quote-selector/quote-selector.styles.js]: Thx! Reverted!

File: src/views/token-swap/components/quote/quote.styles.js (hunk 3 above)
Comment (amonsosanz) [reply src/views/token-swap/components/quote/quote.styles.js:82]: Thx! Reverted!

File: src/views/token-swap/components/quote/quote.styles.js (hunk 8 above)
Comment (amonsosanz) [reply src/views/token-swap/components/quote/quote.styles.js:75]: Thx! Reverted!

File: src/views/token-swap/components/quote/quote.view.jsx (hunk 9 above)
Comment (amonsosanz) [reply src/views/token-swap/components/quote/quote.view.jsx]: Thx! Reverted!

File: src/views/token-swap/components/quote/quote.view.jsx (hunk 4 above)
Comment (amonsosanz) [reply src/views/token-swap/components/quote/quote.view.jsx]: Thx! Reverted!

File: src/views/token-swap/components/selected-quote/selected-quote.style.js (hunk 5 above)
Comment (amonsosanz) [reply src/views/token-swap/components/selected-quote/selected-quote.style.js]: Thx! Reverted!

File: src/views/token-swap/components/selected-quote/selected-quote.style.js (hunk 10 above)
Comment (amonsosanz) [reply src/views/token-swap/components/selected-quote/selected-quote.style.js]: Thx! Reverted!

File: src/views/token-swap/components/selected-quote/selected-quote.view.jsx (hunk 11 above)
Comment (amonsosanz) [reply src/views/token-swap/components/selected-quote/selected-quote.view.jsx]: Thx! Reverted!

File: src/views/token-swap/components/selected-quote/selected-quote.view.jsx (hunk 6 above)
Comment (amonsosanz) [reply src/views/token-swap/components/selected-quote/selected-quote.view.jsx]: Thx! Reverted!

**Review by adrifdez — APPROVED** [review]

**Review by AlbertoElias — APPROVED** [review]

**Pull request merged**